packages/7-todo-tdd/src/components/todo-app/TodoApp.test.tsx

```diff
@@ -1,157 +1,207 @@
+    expect(appTestDriver.getTodosCount()).toBe(0);
+  });
 
-    it('todo input should be rendered', () => {
-        expect(mountedTodoApp.find(h(s.todo__inputContainer__input)).exists()).toBe(true);
-    });
+  it('should add todo item with content when pressing enter', async () => {
+    const content = chance.word();
+    const { appTestDriver } = await givenTodos({});
 
-    it('todo list should be rendered', () => {
-        expect(mountedTodoApp.find(h(s.todo__list)).exists()).toBe(true);
-    });
+    expect(appTestDriver.getTodosCount()).toBe(0);
 
-    it('add btn should be rendered', () => {
-        expect(mountedTodoApp.find(h(s.todo__inputContainer__addBtn)).exists()).toBe(true);
-    });
-});
+    appTestDriver.todoInputInsertContent(content);
+    appTestDriver.pressEnterOnTodoInput();
+    await appTestDriver.waitForAppToUpdate();
 
-describe('checks todo list - add functionality', () => {
-    let appTestDriver: TodoAppDriver;
-
-    beforeEach(() => {
-        appTestDriver = new TodoAppDriver();
-    });
-
-    afterEach(async ()=>{
-        appTestDriver.removeTodosFromFakeTodosService();
-    })
-
-    it('should not add todo item with empty content', async () => {
-        appTestDriver.todoInputInsertContent('');
-        appTestDriver.clickOnAddBtn();
-        await appTestDriver.waitForAppToUpdate();
-        expect(appTestDriver.getTodosCount()).toBe(0);
-    });
-
-    it('should add todo item with content when pressing enter', async () => {
-        appTestDriver.todoInputInsertContent('example');
-        appTestDriver.pressEnterOnTodoInput();
-        await appTestDriver.waitForAppToUpdate()
-        expect(appTestDriver.getTodosCount()).toBe(1);
-    })
-
-    it('should add todo item with content when clicking on add btn', async () => {
-        appTestDriver.todoInputInsertContent('example');
-        appTestDriver.clickOnAddBtn();
-        await appTestDriver.waitForAppToUpdate()
-        expect(appTestDriver.getTodosCount()).toBe(1);
-    })
-
-    it('todo input should be empty after adding todo', async ()=>{
-        appTestDriver.todoInputInsertContent('example');
-        appTestDriver.clickOnAddBtn();
-        await appTestDriver.waitForAppToUpdate()
-        expect(appTestDriver.getTodoInputContent()).toBe(''); 
-    })
-})
+    expect(appTestDriver.getFirstTodo().content).toBe(content);
+  });
+
+  it('should add todo item with content when clicking on add btn', async () => {
+    const content = chance.word();
+    const { appTestDriver } = await givenTodos({});
+
+    expect(appTestDriver.getTodosCount()).toBe(0);
+
+    appTestDriver.todoInputInsertContent(content);
+    appTestDriver.clickOnAddBtn();
+    await appTestDriver.waitForAppToUpdate();
+
+    expect(appTestDriver.getFirstTodo().content).toBe(content);
+  });
+
+  it('todo input should be empty after adding todo', async () => {
+    const { appTestDriver } = await givenTodos({});
+
+    expect(appTestDriver.getTodoInputContent()).toBe('');
```

Comment:
move this line below the line of `todoInputInsertContent` and check that the input is not empty

Reply:
lets discuss it